sim/firesim-lib/src/main/scala/bridges/SimpleNICBridge.scala

```diff
@@ -14,7 +14,7 @@ import icenet.{NICIOvonly, RateLimiterSettings}
 object TokenQueueConsts {
   val TOKENS_PER_BIG_TOKEN = 7
   val BIG_TOKEN_WIDTH = (TOKENS_PER_BIG_TOKEN + 1) * 64
-  val TOKEN_QUEUE_DEPTH = 6144
+  val TOKEN_QUEUE_DEPTH = 3072
```

Comment:
Why is now 1/2 the size?

Reply:
To make it fit by default on nitefury. This still covers the default latency value in config_runtime.yaml